content/_includes/embed.macros.njk

```diff
@@ -309,6 +315,21 @@ params:
 {%- endif -%}
 {%- endmacro %}
 
+{% macro srcset(
+  src,
+  width=none
+) -%}
+  {%- set width = width or 640 -%}
+  {%- set sizes = 'srcset' | fromTaxonomy -%}
+  {%- if width > sizes[0] -%}
+    {%- for size in sizes -%}
+      {%- if size < width -%}
+        {{- src | imgSuffix(size | string) }} {{ size + 'w,' -}}
+      {%- endif -%}
+    {%- endfor -%}
+    {{- src }} {{ width + 'w' -}}
+  {%- endif -%}
+{%- endmacro %}
```

Comment:
@mirisuzanne Is it intentional that we're only using this in `figure()` and `post.hero()` images?

Reply:
I don't know what you mean. `post.hero()` does call `embed.img()`, which applies `srcset()`. I'm seeing srcsets on all relevant images when I look at the deploy preview. What am I missing?

Reply:
We use `embed.img` all over the site. Do you just mean that not all images have the height/width set? I figured this was worth merging before we take the time to update every single image.

Reply:
I've been adding dimension to every image I come across as I work, but we'll need a card to do it more intentionally